Donor/Entities/Organ.cs

```diff
@@ -0,0 +1,8 @@
+namespace Donor.Entities
+{
+    public class Organ
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = string.Empty;
```

Comment:
The relationship with the Donor is missing.
This class should have a collection of Donor and DonorId property